src/awesome/mouse.rs

```diff
@@ -67,15 +67,15 @@ fn coords<'lua>(lua: &'lua Lua, (coords, _ignore_enter): (rlua::Value<'lua>, rlu
             let (x, y) = (coords.get("x")?, coords.get("y")?);
             // TODO The ignore_enter is supposed to not send a send event to the client
             // That's not possible, at least until wlroots is complete.
-            input::pointer::set_position(Point { x, y });
+            input::pointer::set_position_v2(x, y);
             Ok(coords)
         },
         _ => {
             // get the coords
             let coords = lua.create_table();
-            let Point { x, y } = input::pointer::get_position();
-            coords.set("x", x)?;
-            coords.set("y", y)?;
+            let (x, y) = input::pointer::get_position_v2();
+            coords.set("x", x as i32)?;
+            coords.set("y", y as i32)?;
```

Comment:
Hm, according to the [awesome docs](https://awesomewm.org/doc/api/libraries/mouse.html) it uses integers mainly. However this is more "correct" with actually doing what the user wants...

I would say set this as an f64 instead.

Also right now your doing half of each: You return an integer but you (can) take in a floating number. It should return a floating point number.